Watchman.Discord/WatchmanBot.cs

```diff
@@ -53,22 +54,39 @@ public WorkflowBuilder GetWorkflowBuilder()
                             var servers = (await serversService.GetDiscordServers()).ToList();
                             servers.ForEach(unmutingService.UnmuteUsersInit);
                         })
-                        .AddFromIoC<CyclicStatisticsGeneratorService>(cyclicStatsGenerator => async () =>
+                        .AddFromIoC<CyclicStatisticsGeneratorService>(cyclicStatsGenerator => () =>
                         {
                             _ = cyclicStatsGenerator.StartGeneratingStatsCacheEveryday();
+                            return Task.CompletedTask;
                         })
                         .AddFromIoC<ResponsesInitService>(responsesService => async () =>
                         {
                             await responsesService.InitNewResponsesFromResources();
                         })
-                        .AddHandler(() => Task.Run(() => Log.Information("Bot started and logged in...")));
+                        .AddFromIoC<InitializationService, DiscordServersService>((initService, serversService) => () =>
+                        {
+                            var servers = serversService.GetDiscordServers().Result;
+                            Parallel.ForEach(servers, async server =>
```

Comment:
co do używania Parallel polecam posłuchać
https://youtu.be/ueVYdSVsTWc

jakość dźwięku i prowadzenia może nie jest najlepsza
ale można troche z tego wynieść o tym kiedy warto używać wielowątkowości i jak łatwo zepsuć nią projekt